server/routes/starter.js

```diff
@@ -6,4 +6,7 @@ router.get('/', (req, res) => {
   res.send('Welcome to the start of MOA!')
 })
 
+router.get('/', (req, res) => {
+  
```

Comment:
Remove anything to do with starter routes please. This should no longer be in the project :)